apps/builder/app/routes/rest.ai.op.ts

```diff
@@ -5,17 +5,31 @@ import {
   createGptModel,
   type GPTModelMessageFormat,
   createErrorResponse,
+  copywriter,
+  type ModelMessage,
+  copywriterDetect,
 } from "@webstudio-is/ai";
 import { isFeatureEnabled } from "@webstudio-is/feature-flags";
+import { loadBuildByProjectId } from "@webstudio-is/project-build/index.server";
+import { authorizeProject } from "@webstudio-is/trpc-interface/index.server";
+import { collectTextInstances } from "node_modules/@webstudio-is/ai/src/chains/copywriter/chain";
+import { z } from "zod";
 
 import env from "~/env/env.server";
 import { createContext } from "~/shared/context.server";
 
-const RequestSchema = operations.ContextSchema;
+const RequestSchema = operations.ContextSchema.merge(
+  z.object({
+    projectId: z.string(),
+    instanceId: z.string(),
+  })
+);
 
 export const maxDuration = 120;
 
 export const action = async ({ request }: ActionArgs) => {
+  const llmMessages: ModelMessage[] = [];
+
   if (isFeatureEnabled("aiCopy") === false) {
```

Comment:
I think we are launching most of these AI features at once, so probably no reason to separate into separate feature flags